modules/configuring-a-proxy-after-installation-cli.adoc

```diff
@@ -30,17 +30,27 @@ $ rosa edit cluster \
  --cluster $CLUSTER_NAME \
  --additional-trust-bundle-file <path_to_ca_bundle_file> \ <1> <2> <3>
  --http-proxy http://<username>:<password>@<ip>:<port> \ <1> <4>
- --https-proxy http(s)://<username>:<password>@<ip>:<port> <1> <4>
+ --https-proxy http(s)://<username>:<password>@<ip>:<port> \ <1> <4>
+  --no-proxy quay.io <5>
```

Comment:
@EricPonvelle The above example doesn't use real value, I think it would better to use example.com for no_proxy.WDYT?